src/modules/pos/validator/components/ChangeCommission/Form/InputLabel.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Tooltip from 'src/theme/Tooltip';
+import styles from './ChangeCommissionForm.css';
+
+const InputLabel = ({ title, tooltip }) => (
+  <label className={styles.label}>
+    {title}
+    <Tooltip position="right">
```

Comment:
Maybe we should to 
```
    {tooltip && (
      <Tooltip position="right">
        <p>{tooltip}</p>
      </Tooltip>
    )}
 ```
To prevent the above bugg